nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -614,6 +614,11 @@ private static class DefaultLibraryModels implements LibraryModels {
         new ImmutableSetMultimap.Builder<MethodRef, Integer>()
             .put(methodRef("java.util.Optional", "orElse(T)"), 0)
             .put(methodRef("com.google.common.io.Closer", "<C>register(C)"), 0)
+            .put(methodRef("java.util.Map", "getOrDefault(java.lang.Object,V)"), 1)
+            .put(
+                methodRef(
+                    "com.google.common.collect.ImmutableMap", "getOrDefault(java.lang.Object,V)"),
```

Comment:
should we add a comment here why we model ImmutableMap separately?
it might not be immediately clear and relies on `AcknowledgeRestrictiveAnnotations` being set

Reply:
Done in a8b011c486f44a9396bd974fc8c47a6dd679b678